webapp/publisher/snaps/build_views.py

```diff
@@ -24,7 +25,7 @@
 )
 
 
-def get_builds(lp_snap, context, selection):
+def get_builds(lp_snap, context, build_id, selection):
```

Comment:
I think this needs to be refactored so it doesn't modify the context and just returns the required fields that can be added to the context.

Reply:
https://github.com/canonical-web-and-design/snapcraft.io/issues/2550